cmake/tpl/zlib.cmake

```diff
@@ -1 +1 @@
-set(${lib_name}_libs libz.a)
+set(${lib_name}_libs libz.so)
```

Comment:
Why was this changed?

Reply:
There was a problem that I was running into while building a week ago and this was a simple solution to keep working. I need to check it out again.